site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -1,71 +1,110 @@
-  Passing][message-passing] for an overview of the subject. Methods in this category include
-  [connect](/docs/extensions/reference/runtime/#method-connect),
-  [connectNative](/docs/extensions/reference/runtime/#method-connectNative),
-  [sendMessage](/docs/extensions/reference/runtime/#method-sendMessage), and
-  [sendNativeMessage](/docs/extensions/reference/runtime/#method-sendNativeMessage).
+: Your extension can communicate with different contexts within your extension and also with other extensions using these methods and events: 
+[connect][method-connect],
+[onConnect][method-onconnect], 
+[onConnectExternal][method-onconnectexternal],
+[sendMessage][method-sendmessage], 
+[onMessage][method-onconnect] and
+[onMessageExternal][method-onmessageexternal]. 
+In addition, your extension can pass messages to native applications on the user's device using 
+[connectNative][method-connectnative] and
+[sendNativeMessage][method-sendnativemessage]. 
+
+{% Aside %}
+
+See [Message Passing][doc-messages] for an overview of the subject. 
+
+{% endAside %}
 
 Accessing extension and platform metadata
 
 : These methods let you retrieve several specific pieces of metadata about the extension and the
   platform. Methods in this category include
-  [getBackgroundPage](/docs/extensions/reference/runtime/#method-getBackgroundPage),
-  [getManifest](/docs/extensions/reference/runtime/#method-getManifest),
-  [getPackageDirectoryEntry](/docs/extensions/reference/runtime/#method-getPackageDirectoryEntry), and
-  [getPlatformInfo](/docs/extensions/reference/runtime/#method-getPlatformInfo).
+  [getManifest][method-getmanifest], and
+  [getPlatformInfo][method-getplatforminfo].
 
 Managing extension lifecycle and options
 
-: These methods let you perform some meta-operations on the extension, and display the options page
-  to the extension user. Methods in this category include
-  [reload](/docs/extensions/reference/runtime/#method-reload),
-  [requestUpdateCheck](/docs/extensions/reference/runtime/#method-requestUpdateCheck),
-  [setUninstallURL](/docs/extensions/reference/runtime/#method-setUninstallURL), and
-  [openOptionsPage](/docs/extensions/reference/runtime/#method-openOptionsPage).
+: These properties let you perform some meta-operations on the extension, and display the options page. 
+Methods and events in this category include
+  [onInstalled][method-oninstalled],
+  [onStartup][method-onstartup],
+  [openOptionsPage][method-openoptionspage],
+  [reload][method-reload],
+  [requestUpdateCheck][method-requestupdatecheck], and
+  [setUninstallURL][method-setuninstallurl]
 
-Device restart support
+Helper utilities
+
+: These methods provide utility such as the conversion of internal resource representations to
+  external formats. Methods in this category include
+  [getURL][method-geturl].
+
```

Comment:
I don't know what the folks that originally added "helper utilities" had in mind, but I don't have any objections to only having one method in this section for the moment.

It might also be worth noting that with the introduction of dynamic URLs for web accessible resources, this method is the only way for developers to get the URL of a resource.